### ISSUE TITLE
See domains / site reports that do and don't match ISP policy

### ISSUE THREAD

**Issue by edjw** [openrightsgroup/blocked-org-uk#384]
As a non-admin user, I want to filter domains and site reports by:

* an unresolved site report that does match ISP policy
* an unresolved site report that do match ISP policy
* a resolved site report and the block does not match ISP policy

I want to be able to have a link for each of these filters above.

--

This will help users double-check figures in the report and see examples of websites that go through an ISP review process and are still blocked against policy.

--

We are basically taking the data from [this page of the admin area](https://www.blocked.org.uk/control/ispreport/reply-stats) and making it public.

I think it makes most sense to implement this on the Recently reported sites page (https://www.blocked.org.uk/reported-sites). Do you agree @jimkillock and @dantheta?

On that Recently reported sites page, does the title “Incorrectly blocked sites reported to ISPs” make sense to you all? If we add these filters as suggested above, we should definitely change that title as some of the sites in that list do match ISP policy.

**Comment by JimKillock** [comment]
There is a bit of a maintenance issue placing this information in high profile places. We may update this sporadically. So I am inclined to put the stats information in somewhere where this context is more obvious rather than make it look like a live-feed of information.

**Comment by edjw** [comment]
OK, that make sense. 

@dantheta – In that case, I think let's create a new page that provides this information

**Comment by JimKillock** [comment]
I would be inclined to present all of the stats in one place, including the existing stats. I would think about how this should be navigated and understood. For instance, we might make a subdomain with a different navigation system, eg stats.blocked.org.uk rather than trying to integrate the content into an already overcrowded site.

**Comment by dantheta** [comment]
I've made the filters display a bit more concise (but hopefully still quite clear).  The "Incorrectly blocked ..." title changes depending on whether any filters are active.

I've linked the various filtered pages from the main stats page.  Some of the numeric cells are links that re-use the reported-sites page with relevant filters applied.  It's not the most obvious navigation, but it's a step forward.

We could think about a separate stats sub-site.  The spacious layout of the frontend site isn't quite as well suited to dense interactive displays.  An interface similar to a product catalog with a column of filters down the left could be easier to operate and would leave more space for the data displays.
